networkx/algorithms/tests/test_steinertree.py

```diff
@@ -0,0 +1,42 @@
+from nose.tools import assert_equal
+import networkx as nx
+from networkx.algorithms.steinertree import metric_closure, steiner_tree
+
+
+class TestSteinerTree:
+    def set_up(self):
```

Comment:
@NareshPeshwe You can not define "set up" for a test class as `def set_up`. Go with something like `def setup` or `def setUp`.

From nosetests documentation.

> To create package-level setup and teardown methods, define setup and/or teardown functions in the __init__.py of a test package. Setup methods may be named setup, setup_package, setUp, or setUpPackage; teardown may be named teardown, teardown_package, tearDown or tearDownPackage. Execution of tests in a test package begins as soon as the first test module is loaded from the test package.

http://nose.readthedocs.io/en/latest/writing_tests.html